python/sglang/srt/managers/scheduler.py

```diff
@@ -709,8 +698,8 @@ def update_running_batch(self):
             self.waiting_queue.extend(retracted_reqs)
         else:
             self.new_token_ratio = max(
-                self.new_token_ratio - self.new_token_ratio_decay,
-                self.min_new_token_ratio,
+                self.new_token_ratio - global_config.new_token_ratio_decay,
+                global_config.min_new_token_ratio,
```

Comment:
why this change?
`global_config` is supposed to be a ready-only variable after initialization.